src/main/java/org/quantumbadger/redreader/reddit/prepared/html/HtmlRawElement.java

```diff
@@ -269,7 +269,9 @@ public static HtmlRawElement readFrom(@NonNull final HtmlReaderPeekable reader)
 					break;
 
 				case "img":
-					result = new HtmlRawElementPlainText("Image");
```

Comment:
I think this might break GIFs in comments? Currently these are shown as a link containing this "Image" plain text element.

Reply:
I don't think it breaks gifs. I've been using my build for a few weeks now and haven't noticed anything. I'll double check just in case

Reply:
GIFs seem to work fine. The only side effect I've seen is that a preview of the GIF is now shown above the link, which is probably more of a feature than a bug

I'm now passing emotes with a custom `emote` tag just in case though. If we discover any bugs or users don't like the GIF preview, we can always revert this case condition in `HtmlRawElement.java`.

``` 
case "img":
	return new HtmlRawElementImg(children,
	        startToken.title == null || startToken.title.isEmpty()
			        ? "emote" : startToken.title,
	        startToken.src);
```
This will resolve it without affecting the subreddit emotes